src/main.rs

```diff
@@ -81,6 +81,13 @@ fn main() -> Result<()> {
                         )
                         .takes_value(true),
                 )
+                .arg(
+                    Arg::new("current-dir")
+                        .long("current-dir")
+                        .help("Output Nixpacks related files to the current directory ")
+                        .takes_value(false)
+                        .global(true),
```

Comment:
This flag doesn't need to be globally and should only be available for the `build` command